python-backend/app/api/incidents/models/mine_incident.py

```diff
@@ -21,12 +21,10 @@ class MineIncident(AuditMixin, Base):
     __tablename__ = 'mine_incident'
 
     mine_incident_id = db.Column(db.Integer, primary_key=True, server_default=FetchedValue())
-    mine_incident_id_year = db.Column(db.Integer,
-                                      nullable=False,
-                                      default=datetime.datetime.now().year)
-    mine_incident_guid = db.Column(UUID(as_uuid=True),
-                                   nullable=False,
-                                   server_default=FetchedValue())
+    mine_incident_id_year = db.Column(
```

Comment:
Looks like we might have some conflicting python formatters by different developers. We should probably standardize on one.

Reply:
Yes please